### DISCUSSION TITLE
Would it be ok for me to split up the python files more.

### DISCUSSION THREAD

**Discussion by NeeEoo** [elikaski/BF-it#45] [Q&A]
I've noticed that importing a file sometimes causes a circular dependency error. So this will also fix it.

**Comment by elikaski** [comment]
Sure
Before you start, can you roughly say which changes will be made?

**Reply by NeeEoo** [reply]
Moved LibraryFunctionCompiler into a separate file and moved get_readint_code, get_printint_code, get_readchar_code, get_printchar_code into the same file.
Moved insert_function_object, get_function_object, check_function_exists, and the global dict containing the functions.
Moved both Error classes into its own file.

Replacing the `from x import *` to `from x import y, z`

Replacing `from Compiler.Token import Token` with `from .Token import Token` (Removing the Compiler part of the import)

**Reply by elikaski** [reply]
Sounds good

**Reply by NeeEoo** [reply]
What would you like the commit name to be? I can't come up with one that sounds good

**Reply by elikaski** [reply]
How about "divide x,y,z files into smaller modules" ?

**Reply by NeeEoo** [reply]
"Split some files into smaller modules"?
